generators/php.js

```diff
@@ -149,14 +149,23 @@ Blockly.PHP.init = function(workspace) {
     Blockly.PHP.variableDB_.reset();
   }
 
+  Blockly.PHP.variableDB_.setVariableMap(workspace.getVariableMap());
+
   var defvars = [];
-  var varName;
-  var variables = Blockly.Variables.allVariables(workspace);
+  var variables = workspace.getAllVariables();
   for (var i = 0, variable; variable = variables[i]; i++) {
-    varName = variable.name;
-    defvars[i] = Blockly.PHP.variableDB_.getName(varName,
+    defvars[i] = Blockly.PHP.variableDB_.getName(variable.getId(),
         Blockly.Variables.NAME_TYPE) + ';';
   }
+
+  // Add developer variables (not created or named by the user).
```

Comment:
A lot of the developer variable stuff in generator stuff is pretty repetitive. e.g. this loop is almost the same in all of them. Is there a way to avoid that?

Reply:
Probably, but I'm punting on that API cleanup for this PR.  Here's a bug for it: https://github.com/google/blockly/issues/1535